test/testdata/expected/TestArray.go

```diff
@@ -0,0 +1,96 @@
+package expected
+
+var TestArrayStructGo string = `
+type TestArray struct {
+	c C.TestArray
+}`
+
+var TestStructsGetter string = `
+func (ta TestArray) Structs() []EmptyStruct {
+	var s []EmptyStruct
+	gos_s := (*reflect.SliceHeader)(unsafe.Pointer(&s))
+	gos_s.Cap = int(10)
+	gos_s.Len = int(10)
+	gos_s.Data = uintptr(unsafe.Pointer(ta.c.structs))
+
+	return s
+}`
+
+var TestFixedSizedArrayGetter string = `
+func (ta TestArray) FixedSizedArray() []uint64 {
+	var s []uint64
+	gos_s := (*reflect.SliceHeader)(unsafe.Pointer(&s))
+	gos_s.Cap = int(10)
+	gos_s.Len = int(10)
+	gos_s.Data = uintptr(unsafe.Pointer(ta.c.fixedSizedArray))
+
+	return s
+}`
+
+/*var TestInitArrayGetter string = `
+	func (TestArray ta) InitArray() []uint{
+		var s []uint
+		gos_s := (*reflect.SliceHeader)(unsafe.Pointer(&s))
+		gos_s.Cap = int(3)
+		gos_s.Len = int(3)
+		gos_s.Data = uintptr(unsafe.Pointer(ta.c.initArray))
+
+		return s
+	}
+`*/
+
+var TestFunctionWithStructArrayParam = `
+func (ta TestArray) FunctionWithStructArrayParam(earr [10]EmptyStruct) {
```

Comment:
I guess all of these outcomes will change because of https://github.com/go-clang/gen/pull/128 ?